blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -381,12 +381,9 @@ class ResultsList extends Component {
           className="btn btn-sm"
           primaryFont={getThemeStyle(arcSite)['primary-font-family']}
           primaryColor={getThemeStyle(arcSite)['primary-color']}
+          aria-label={this.phrases.t('results-list-block.see-more-button-aria-label')}
         >
           {this.phrases.t('results-list-block.see-more-button')}
-          {' '}
```

Comment:
this was removed?

Reply:
It was moved to be in an `aria-label` to allow language strings to not need special formatting